### PR TITLE
fix map init zoom

### PR DESCRIPTION
**Pull request by JeremyLi28**
Fully abandon the angular-leaflet-directive needs some effort, here is a quick fix

### CODE REVIEW

File: neo/app/assets/javascripts/map/controllers.js [hunk 1]
```diff
@@ -65,6 +65,7 @@ angular.module('cloudberry.map', ['leaflet-directive', 'cloudberry.common'])
       leafletData.getMap().then(function(map) {
         $scope.map = map;
         $scope.bounds = map.getBounds();
+        map.setView([39.5, -96.35],4);
```
Comment (JavierJia) [inline neo/app/assets/javascripts/map/controllers.js]: could you make this center and level as two directive attribute ?

Comment (JeremyLi28) [inline neo/app/assets/javascripts/map/controllers.js]: sure

Comment (JeremyLi28) [inline neo/app/assets/javascripts/map/controllers.js]: Done

**Comment by JavierJia** [comment]
LGTM 👍

**Comment by JavierJia** [comment]
@JeremyLi28 you can go ahead and merge it.

**Comment by JeremyLi28** [comment]
It seems that I don't have the privilege to merge?

**Comment by JavierJia** [comment]
updated the team mode to write. You will be able to merge now.

**Comment by JeremyLi28** [comment]
Cool!

**Pull request merged**